packages/react/src/components/Hero/Hero.tsx

```diff
@@ -18,7 +18,7 @@ const HeroImage: FC<ImageProps> = ({ url, alt }) => {
   const defaultSrc = sortedUrls[sortedUrls.length - 1].src;
 
   return (
-    <figure className="hero--figure">
+    <figure className="hero__row--with-image hero--figure">
```

Comment:
An element should not have multiple BEM element annotations, in this case, it is both hero__row and hero--figure. You can either add row to another element or add hero--figure__no-image for example.

Also pay attention to the separators, this project uses a modified BEM convention as noted in the outline - https://docs.google.com/document/d/18p5Z_d2gPE3gNWaVW5Ti6N65j8RR8Y0BqAdKEQt-yjk/edit?pli=1#heading=h.go781menmhq0

Reply:
Overall, I still think it would be better if we can separate the concerns and render a custom background component instead of the image component when no image is present.

Reply:
oh got it now @bashlk will do the changes :)